tests/config/nextflow.config

```diff
@@ -22,5 +22,5 @@ if ("$PROFILE" == "singularity") {
 }
 
 manifest {
-  nextflowVersion = '!>=20.11.0-edge'
+  nextflowVersion = '!>=20.10.0'
```

Comment:
```suggestion
  nextflowVersion = '!>=20.11.0-edge'
```

Reply:
Need to revert this commit @yuukiiwa We need that specific version in this file.

Reply:
Thanks, @drpatelh! I have reverted it back to `!>=20.11.0-edge`